internal/controllers/topology/cluster/patches/variables/variables.go

```diff
@@ -33,6 +33,8 @@ import (
 const (
 	// BuiltinsName is the name of the builtin variable.
 	BuiltinsName = "builtin"
+	// noConflictsDefinitionFrom is the value of the variable DefinitionFrom field when a variable definition no conflicts.
```

Comment:
I think this is not true, even when a variable doesn't have a conflict users can still set a DefinitionsFrom